cli/src/semgrep/core_runner.py

```diff
@@ -463,19 +475,34 @@ def execute(self) -> int:
 class Task:
     path: str = field(converter=str)
     language: Language
-    rule_nums: Sequence[int]
+    # a rule_num is the rule's index in the rule ID list
+    rule_nums: Tuple[int, ...]
+
+    @property
+    def language_label(self) -> str:
+        return (
+            "<multilang>"
+            if self.language in {Language("regex"), Language("generic")}
+            else self.language
+        )
 
 
 class TargetMappings(List[Task]):
     @property
     def rule_count(self) -> int:
-        return len({rule for task in self for rule in task.rule_nums})
+        return len({rule_num for task in self for rule_num in task.rule_nums})
```

Comment:
len of a rule_num seems wrong

Reply:
Yeah it was poorly named. rule_num is actually rule_index. But I had to stop myself from refactoring more code because the PR was already huge 😅